app/src/main/java/com/tinkoff/edu/test/LoanCalcTest.java

```diff
@@ -1,14 +1,18 @@
 package com.tinkoff.edu.test;
 
+import com.tinkoff.edu.app.ClientType;
 import com.tinkoff.edu.app.LoanCalcController;
+import com.tinkoff.edu.app.LoanRequest;
 
 /**
  * Loan Calc Tests
  */
 public class LoanCalcTest {
     public static void main(String... args) {
 
-        int requestId = LoanCalcController.createRequest();
-        System.out.println(requestId + " should be 1");
+        System.out.println(
+                new LoanCalcController().createRequest(
+                        new LoanRequest(ClientType.PERSON,10,1000))
```

Comment:
лучше писать как принято) пробел после знаков препинания
```suggestion
                        new LoanRequest(ClientType.PERSON, 10, 1000))
```

Reply:
в идее там подставлены подсказки и не видно, что пробелов нет))